src/main/tut/index.html

```diff
@@ -445,6 +488,7 @@
 --
 
 We have `Either` to represent these cases:
+<!-- COMMENT mention biasing? -->
```

Comment:
Valid point. I feel that whether or not `Either` has a monad here isn't hugely relevant to the point (implicit composition of type class instances), but I could certainly mention it orally and let people know that better alternatives exist.

As a side note, I believe `Either` is actually right-biased now, and that cats dropped `Xor` in its favour.